addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -295,7 +295,8 @@
             <a t-att-href="'/slides/%s/tag/%s' % (slug(slide.channel_id), slug(tag))" class="badge text-bg-info py-1 px-2" t-esc="tag.name"/>
         </t>
     </div>
-    <div class="oe_structure oe_empty"/>
+    <div class="oe_structure oe_empty" id="oe_structure_website_slides_lesson_top_1"
```

Comment:
Can you explain me in which case the id is necessary ?
What do you mean when the view is updated ?